examples/generative/md/stylegan.md

```diff
@@ -727,26 +749,16 @@ train(start_res=4, target_res=16, steps_per_epoch=1, display_images=False)
 ```
 
     
-<div class="k-default-codeblock">
 ```
 Model resolution:4x4
 STABLE
 1/1 [==============================] - 3s 3s/step - d_loss: 2.0971 - g_loss: 2.5965
-```
-</div>
-    
-<div class="k-default-codeblock">
-```
 Model resolution:8x8
 TRANSITION
 1/1 [==============================] - 5s 5s/step - d_loss: 6.6954 - g_loss: 0.3432
 STABLE
 1/1 [==============================] - 4s 4s/step - d_loss: 3.3558 - g_loss: 3.7813
-```
-</div>
-    
-<div class="k-default-codeblock">
-```
+``
```

Comment:
It seems there are a few glitches with these files. Did you generate them with `python autogen.py add_example generative/stylegan`?

Reply:
Yes, but the script output (`stylegan.md`) had error notices which I felt were distracting and tried to remove manually.

For example: `WARNING:tensorflow:Inconsistent references when loading the checkpoint`

I've uploaded the output of the `autogen.py` script as gists for your review:
- [stylegan.md](https://gist.github.com/yarri-oss/72ac246a7063a13db2a8be056b7ac5f8)
- [stylegan.ipynb](https://gist.github.com/yarri-oss/4dabc844b11fa5059d43ad2bdaa24c64)